### PR TITLE
Update tutorials with `"dry-run"`

### PR DESCRIPTION
**Pull request by bharat-thotakura**
Addresses most of https://github.com/Infleqtion/client-superstaq/issues/696 by allowing free-trial users to run the notebooks for now. 

The qss notebook requires https://github.com/Infleqtion/server-superstaq/pull/2631

### CODE REVIEW

File: docs/source/get_started/basics/basics_css.ipynb [hunk 1]
```diff
@@ -138,12 +138,16 @@
    "metadata": {},
    "source": [
     "## Submit your circuit and view results\n",
-    "Finally, we can submit our circuit to the desired device and view the results of our job. Here, we use [IBM's QASM simulator](https://quantum-computing.ibm.com/lab/docs/iql/manage/simulator), but you can access many other devices via Superstaq!"
+    "Finally, we can submit our circuit to the desired device (by specifying `target=`) and view the results of our job. \n",
+    "\n",
+    "For the purposes of this tutorial, we will instruct Superstaq to simulate the circuit to the desired device backend by providing the `\"dry-run\"` method parameter, a feature that is available to all users including free-trial users. \n",
+    "\n",
+    "Here, we use [IBM's QASM simulator](https://quantum-computing.ibm.com/lab/docs/iql/manage/simulator), but you can access many other devices via Superstaq!"
```
Comment (epelaaez) [inline docs/source/get_started/basics/basics_css.ipynb:145]: > Here, we use [IBM's QASM simulator](https://quantum-computing.ibm.com/lab/docs/iql/manage/simulator)

Should we still say this if it isn't really submitting to the IBM simulator? Or expand on 'dry-run' saying that it is a simulation done by us.

Comment (bharat-thotakura) [reply docs/source/get_started/basics/basics_css.ipynb:145]: I think 

> we will instruct Superstaq to simulate the circuit to the desired device backend by providing the `\"dry-run\"` method 

communicates that the simulation is done by Superstaq, but I think you are right that I should clarify that we're running this simulation _to the target_/_based on_ [IBM's QASM simulator](https://quantum-computing.ibm.com/lab/docs/iql/manage/simulator). Perhaps there's better wording for all this? 

cc @stephanielee9 @vtomole

Comment (epelaaez) [reply docs/source/get_started/basics/basics_css.ipynb:145]: I think even just switching the order of this statements would make it clearer. So something like:

> Here, we use [IBM's QASM simulator](https://quantum-computing.ibm.com/lab/docs/iql/manage/simulator), but you can access many other devices via Superstaq!
> 
> For the purposes of this tutorial, we will instruct Superstaq to simulate the circuit in-house to the desired device backend by providing the `\"dry-run\"` method parameter, a feature that is available to all users including free-trial users.

(with better wording on the second paragraph)

Comment (bharat-thotakura) [reply docs/source/get_started/basics/basics_css.ipynb:145]: That sounds good thanks, will update

**Review by epelaaez — APPROVED** [review]
lgtm!

**Pull request merged**